src/Native/Runtime/RhConfig.cpp

```diff
@@ -51,13 +51,13 @@ UInt32 RhConfig::ReadConfigValue(_In_z_ const WCHAR *wszName, UInt32 uiDefaultVa
     {
         uiResult <<= 4;
 
-        WCHAR ch = wszBuffer[i];
-        if ((ch >= L'0') && (ch <= L'9'))
-            uiResult += ch - L'0';
-        else if ((ch >= L'a') && (ch <= L'f'))
-            uiResult += (ch - L'a') + 10;
-        else if ((ch >= L'A') && (ch <= L'F'))
-            uiResult += (ch - L'A') + 10;
+        TCHAR ch = wszBuffer[i];
+        if ((ch >= '0') && (ch <= '9'))
```

Comment:
Mixing of char literals and TCHAR variables makes me a bit uneasy.  It's probably being over-paranoid, but should we be using a macro wrapper around the literals?